sklearn/utils/tests/test_validation.py

```diff
@@ -1200,7 +1200,7 @@ def test_check_fit_params(indices):
 def test_check_sparse_pandas_sp_format(sp_format):
     # check_array converts pandas dataframe with only sparse arrays into
     # sparse matrix
-    pd = pytest.importorskip("pandas")
+    pd = pytest.importorskip("pandas", minversion="0.25.0")
```

Comment:
sparse support was added in `0.25` and the pandas version installed on 18.04 is `0.22.0`.